src/NodeJS/Server.php

```diff
@@ -254,6 +262,52 @@ public function getConnection()
     }
 
     /**
+     * Return the options.
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->options;
+    }
+
+    /**
+     * Set options array.
+     */
+    public function setOptions($options)
```

Comment:
Please update example in README.md to show that it's now possible to set Zombie options.

While we're at it you can send PR's to MinkExtension and PHPUnit-Mink to update Zombie integration to allow specifying Zombie options in there as well.

That would be a great addition.